app/src/main/res/values/strings.xml

```diff
@@ -14,7 +14,7 @@
 <string name="position_long"><b>longitude</b></string>
     <string name="placeholder_position_long">2.17403</string>
     <string name="email"><b>Email</b></string>
-    <string name="placeholder_email">exemple@mail.com</string>
+    <string name="placeholder_email">example@mail.com</string>
     <string name="password"><b>Password</b></string>
     <string name="sign_up">Create account</string>
     <string name="already_account"><b>Already have an account ? </b></string>
```

Comment:
"information" instead of "informations" line 27 cause information is invariable in english  + line 29 upper case for "write" → "Write"